pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -541,6 +541,19 @@ func Test_InFlightPods(t *testing.T) {
 				q.Add(logger, p)
 			}
 
+			// When a Pod is added to the queue, the QueuedPodInfo will have a new timestamp.
+			// On Windows, time.Now() is not as precise, 2 consecutive calls may return the same timestamp.
+			// Thus, all the QueuedPodInfos can have the same timestamps, which can be an issue
+			// when we're expecting them to be popped in a certain order (the Less function
+			// sorts them by Timestamps if they have the same Pod Priority).
+			// Artificially adding a delta will solve this issue for Windows unit test runs.
+			delta := time.Duration(time.Second)
+			for _, obj := range q.activeQ.List() {
+				pInfo := obj.(*framework.QueuedPodInfo)
+				pInfo.Timestamp = pInfo.Timestamp.Add(delta)
+				delta += time.Second
```

Comment:
For the sake of enable this testcase on windows, I am fine with it, might be you can link to this issue again here (https://github.com/golang/go/issues/8687) and drop a note to remove the workaround here if the issue could be fixed on windows on day.

/lgtm

/cc @kubernetes/sig-scheduling-misc  someone from the list can take a look and approve.

Reply:
The suggestion still makes sense I think.

Reply:
+1, prefer to use a fake clock.